src/Components/SortControl/SortControl.css

```diff
@@ -0,0 +1,40 @@
+.sort-control {
+  font-size: 14px;
+  font-weight: 400;
+  letter-spacing: 0.89px;
+  display: flex;
+  flex-direction: row;
+}
+
+.sort-control label {
+  opacity: 0.6;
+  text-transform: uppercase;
+  margin-right: 20px;
+}
+
+.sort-control .custom-select {
+  display: inline-block;
+  display: grid;
+  grid-template-areas: "select";
+  align-items: center;
+  width: 130px;
+}
+
+.sort-control .custom-select::after {
+  content: "";
+  width: 0.6em;
+  height: 0.4em;
+  background-color: #F65261;
+  clip-path: polygon(100% 0%, 0 0%, 50% 100%);
+  justify-self: end;
+}
+
+.sort-control .custom-select select {
+  text-transform: uppercase;
+  color: #fff;
```

Comment:
It's better to create separate file with variables for colors/fonts/etc

Reply:
Added several variables to index.css for font-size and color.